example/src/main/config/qpid/qdrouterd-with-broker.json

```diff
@@ -118,6 +119,34 @@
       }
   }],
 
+  ["vhost", {
+    "id": "hono-mqtt-internal",
+    "maxConnections": 30,
+    "groups": {
+      "Hono": {
+        "users": "Eclipse IoT;Hono;mqtt-adapter",
+        "remoteHosts": "*",
+        "allowUserIdProxy": true,
```

Comment:
what about maxFrameSize here? Would that allow an application to send a full video e.g. upstream to the device?

Reply:
I have no idea how big the data will be. For telemetry and events I would not expect very big messages but for commands I can image e.g. a firmware update.